### ISSUE TITLE
Could not install from ethereumjs-abi as it does not contain a package.json file.

### ISSUE THREAD

**Issue by StillFantastic** [MetaMask/eth-sig-util#135]
When I run `npm install`, I got and error message:
```
npm ERR! Could not install from "node_modules/eth-sig-util/ethereumjs-abi@git+https:/github.com/ethereumjs/ethereumjs-abi.git" as it does not contain a package.json file.
```
In my `package-lock.json`'s `eth-sig-util` section, I see:
```
    "eth-sig-util": {
      "version": "1.4.2",
      "resolved": "https://registry.npmjs.org/eth-sig-util/-/eth-sig-util-1.4.2.tgz",
      "integrity": "sha1-jZWCAsftuq6Dlwf7pvCf8ydgYhA=",
      "requires": {
        "ethereumjs-abi": "ethereumjs-abi@git+https://github.com/ethereumjs/ethereumjs-abi.git",
        "ethereumjs-util": "^5.1.1"
      },
```
It works If I delete `ethereumjs-abi@` before `git+https`, but it shows up again if I run `npm install`, not sure if it is a bug?

**Comment by just-toby** [comment]
this is a duplicate of #134

**Comment by Gudahtt** [comment]
Thanks for the report! Unfortunately we don't support v1.x anymore. Please update to v2 or v3 - this problem should be fixed in both v2.5.4 and v3.0.1.
